thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/constant/AnomalyResultSource.java

```diff
@@ -0,0 +1,6 @@
+package com.linkedin.thirdeye.constant;
+
+public enum AnomalyResultSource {
```

Comment:
We'll need a value for user-reported anomalies as well

Reply:
This user-reported anomalies will be in a separate table. I would ask Yen about his design on this part.

Reply:
Resolved